examples/language/test/records_test.dart

```diff
@@ -106,8 +106,7 @@ void main() {
     ({int x, int y, int z}) point = (x: 1, y: 2, z: 3);
     ({int r, int g, int b}) color = (r: 1, g: 2, b: 3);
 
-    // Compile error! These record don't have the same shape.
-    // point = color;
+    print(point == color); // Prints 'false'. Lint: Equals on unrelated types.
```

Comment:
Except that the lint is disabled?

Ah, perhaps the point is that the documentation web page should have the comment (because the documentation web page doesn't say anything about `// ignore...`, and we want to tell the reader that they'd get that lint. And `records_test.dart` should contain the comment because it must be identical (in some sense) to the code that occurs on the documentation web page. So it's all working as intended. ;-)